app/src/main/kotlin/com/wire/android/WireApplication.kt

```diff
@@ -43,7 +58,36 @@ class WireApplication : Application(), Configuration.Provider {
     override fun onCreate() {
         super.onCreate()
         if (this.isGoogleServicesAvailable()) {
-            FirebaseApp.initializeApp(this)
+            val serverConfig: ServerConfig = ServerConfig.DEFAULT
+
+            when (serverConfig.title.lowercase()) {
+                ServerTitle.STAGING.title, ServerTitle.QA_DEMO.title,
+                ServerTitle.ANTA.title, ServerTitle.BELLA.title,
+                ServerTitle.CHALA.title -> {
+                    FirebaseApp.initializeApp(
+                        this, FirebaseOptions.Builder()
+                            .setProjectId(FB_STAGING_PROJECT_ID)
+                            .setApplicationId(FB_STAGING_APPLICATION_ID)
+                            .setApiKey(FB_STAGING_API_KEY)
+                            .setGcmSenderId(FB_STAGING_SENDER_ID)
+                            .build()
+                    )
+                }
+                else -> {
+                    FirebaseApp.initializeApp(
+                        this, FirebaseOptions.Builder()
+                            .setProjectId(FB_PROD_PROJECT_ID)
+                            .setApplicationId(FB_PROD_APPLICATION_ID1)
+                            .setApplicationId(FB_PROD_APPLICATION_ID2)
+                            .setApplicationId(FB_PROD_APPLICATION_ID3)
+                            .setApplicationId(FB_PROD_APPLICATION_ID4)
+                            .setApplicationId(FB_PROD_APPLICATION_ID5)
```

Comment:
Wondering why we are passing 5 IDs here

Reply:
for all the projects that we have in the firebase console